commands/deploy_website.js

```diff
@@ -1,226 +1,260 @@
-        type: 'string'
-    },
-
-    endpoint: {
-        describe: 'Endpoint is the URL of a welcome node to receive the transaction',
-        demandOption: true, // Required
-        type: 'string'
-    },
-
-    folder: {
-        describe: 'Folder is the name of folder',
-        demandOption: true, // Required
-        type: 'string'
-    }
-}
-
-exports.handler = async function (argv) {
-    function ReadDirectory(Directory) {
-        try {
-            fs.readdirSync(Directory).forEach(File => {
-                const abs = path.join(Directory, File);
-                if (fs.statSync(abs).isDirectory()) return ReadDirectory(abs);
-                else return Files.push(abs);
-            });
-        } catch (e) {
-            console.error(chalk.red(e.message))
-            return
-        }
-    }
-
-    ReadDirectory(argv.folder);
 
-    for (let i = 0; i < Files.length; i++) {
-        const hmac = crypto.createHmac(algo, argv.seed);
-        hmac.update(Files[i])
-        const seed = hmac.digest('hex')
-        Seed.push(seed)
-        const address = archethic.deriveAddress(seed, 0)
-        Address.push(address)
-    }
+exports.command = "deploy-website";
 
-    tx = archethic.newTransactionBuilder("transfer")
+exports.describe =
+  "Deploy all files inside folder and target index.html file to automatically convert filepaths to transactions";
 
-    for (let i = 0; i < Address.length; i++) {
-        tx.addUCOTransfer(Address[i], 1.0)
-    }
+exports.builder = {
+  seed: {
+    describe:
+      "Seed is a string representing the transaction chain entropy to be able to derive and generate the keys for the transactions",
+    demandOption: true, // Required
+    type: "string",
+  },
+
+  endpoint: {
+    describe:
+      "Endpoint is the URL of a welcome node to receive the transaction",
+    demandOption: true, // Required
+    type: "string",
+  },
+
+  folder: {
+    describe: "Folder is the name of folder",
+    demandOption: true, // Required
+    type: "string",
+  },
+};
 
-    txn = tx
-        .build(argv.seed, 0)
-        .originSign(originPrivateKey)
+exports.handler = async function (argv) {
+  const path_struct = {};
+  const node_endpoint = argv.endpoint;
 
+  function ReadDirectory(Directory) {
     try {
-        await archethic.sendTransaction(txn, argv.endpoint)
+      fs.readdirSync(Directory).forEach((File) => {
+        const abs = path.join(Directory, File);
+
+        if (fs.statSync(abs).isDirectory()) return ReadDirectory(abs);
+        else {
+          if (Directory !== argv.folder) {
+            const key = Directory.replace(argv.folder + "/", "") + "/" + File;
+            path_struct[key] = {
+              seed: "",
+              address: "",
+              full_file_path: abs,
+            };
+            return;
+          } else {
+            path_struct[File] = {
+              seed: "",
+              address: "",
+              full_file_path: abs,
+            };
+            return;
```

Comment:
You can abstract this to a function and re-use it as instead repeating
```suggestion
            function pathStructure(filePath) {
              {
                seed: "",
                address: "",
                full_file_path: filePath
              }
             }
```
and just call something like `path_struct[key] = pathStructure(abs)` 

what do you think? :)

Reply:
Sure thanks, Applied it.

Reply:
Sure thanks, Applied it.